publication/models/distribution_list.py

```diff
@@ -113,6 +113,7 @@ def _compute_counts(self):
     available_count = fields.Integer(
         string="Number still available",
         compute='_compute_counts')
+    analytic_invoice_line_id = fields.Many2one('account.analytic.invoice.line')
```

Comment:
This should not be here, as the total number of copies for a publication "contracted" can be from multiple contract lines, or even multiple contracts. This is because in some cases an organisation gets a number of free copies (for instance included in membership), but can purchase additional copies.